src/clientdlg.cpp

```diff
@@ -858,6 +858,16 @@ void CClientDlg::OnNumClientsChanged ( int iNewNumClients )
     SetMyWindowTitle ( iNewNumClients );
 }
 
+void CClientDlg::OnOpenGeneralSettings()
```

Comment:
Sorry to be a pain.  If this is `SETTING_TAB_AUDIONET`, can the slot be called `OnOpenAudioNetSettings`, please, for consistency.  Alternatively, change the name of the defined value.